lib/Apache/ConfigParser/Directive.pm

```diff
@@ -1071,6 +1116,37 @@ sub directive_value_is_not_dev_null_and_pipe_and_syslog {
   return $_[0] !~ /^\s*(?:(?:\|)|(?:syslog(?::[a-zA-Z0-9]+)?))/;
 }
 
+sub dump {
+    my $node = shift;
+    my $level = shift || 0;
+    my $content = '';
+    my @ds = $node->daughters;
+    if( @ds ) {
+        $content .= '  ' x $level;
```

Comment:
I would prefer to see an array built up and then concatenate it at the end instead of appending to a string continuously, since the later is less efficient.